src/backend/joanie/core/admin.py

```diff
@@ -60,14 +60,42 @@ def owner(self, obj):  # pylint: disable=no-self-use
         return obj.order.owner
 
 
-class CourseProductRelationInline(admin.TabularInline):
+class CourseProductRelationInline(admin.StackedInline):
```

Comment:
If I understand, the PR is about the relation between Course and CourseRun, why there is a change on the representation between Course and Product ? Is that change made on purpose ?

Reply:
That's because I was working to improve the layout of the admin course change view, then I find more convenient to use a StackedInline class to display product relation.